it/endpoint/UpdateCaseEndpointSpec.scala

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package endpoint
+
+import data.SampleData
+import play.api.http.Status
+import play.api.libs.ws.{WSRequest, WSResponse}
+import stubs.{AuditStub, AuthStub, FileTransferStub, UpdateCaseStub}
+import support.IntegrationSpec
+
+class UpdateCaseEndpointSpec extends IntegrationSpec with SampleData {
+
+  "calling the update case endpoint with a valid payload" should {
+
+    "return an OK response" in {
+
+      AuditStub.audit()
+      AuthStub.authorised()
+      UpdateCaseStub.success()
+      FileTransferStub.success()
+
+      val request: WSRequest = buildRequest("/update-case")
+
+      val response: WSResponse = await(request.post(updateCaseJson))
+
+      response.status shouldBe Status.OK
+
+    }
+
```

Comment:
can we have a test for returning a bad_request with the error code and message? I know this is missing from create case but no need to fix that as it is not this story.

Reply:
done